configure.ac

```diff
@@ -531,14 +532,28 @@ AS_VERSION_COMPARE(${CASS_DRV_VERSION}, "2.2", [CASS_DRV_V22=0], [CASS_DRV_V22=1
 AS_IF([test "$CASS_DRV_V22" = 1],
       AC_DEFINE([IDX_CASS_DRV_V22]))
 
+# user-mode {{{3
+AC_ARG_WITH([user-mode],
+        [AS_HELP_STRING([--with-user-mode],
```

Comment:
`--with-user-mode` is a bit confusing name, because we always do with-user-mode, don't we?

I'd recommend naming it: `--with-user-mode-only`.

Reply:
Agree, updated the changes `--with-user-mode-only`.